app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -605,7 +611,7 @@
 
 
     <!-- Sample text for accessibility font size -->
-    <string name="accessibility_text_size_sample_text">The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog.
+    <string name="accessibility_text_size_sample_text">The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog. 
```

Comment:
Nit: can we use two differing pangrams, instead of repeating the same one twice?

> The five boxing wizards jump quickly.
> Pack my box with five dozen liquor jugs.
> How quickly daft jumping zebras vex.

More seriously, I note that the this appears in the `zh-rCN` strings file, but isn't translated.